### PR TITLE
Update volume mount path to read from correct PVC path

### LINKED ISSUE #295: [KFNBC]: No persistence configured
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

No persistent storage has been configured. So closing the notebook server means you lose all the content you might have created. 

### Expected Behavior

I'd expect to find my files again after stopping and restarting my notebook. 

### Steps To Reproduce

1. log in
2. create a notebook file.
3. see it saved, (in the file navigation on the left)
4. close the notebook server
5. reopen a new notebook server
6. your file is no longer there. 

### Workaround (if any)

_No response_

### OpenShift Version

_No response_

### Openshift Version

_No response_

### What browsers are you seeing the problem on?

_No response_

### Open Data Hub Version

_No response_

### Relevant log output

_No response_

### PR DESCRIPTION
**Pull request by DaoDaoNoCode**
Closes #295

### CODE REVIEW

**Comment by samuelvl** [comment]
@DaoDaoNoCode lgtm but you need 2 reviewers. Can you add the new `notebook-controller` label to this issue (and future ones) and assign it to the https://github.com/orgs/opendatahub-io/projects/15#card-79500415

**Comment by samuelvl** [comment]
@DaoDaoNoCode New changes tested:

image

Can you add one more reviewer to get this merged?

**Review by maroroman — APPROVED** [review]

File: frontend/src/pages/notebookController/const.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 export const CUSTOM_VARIABLE = 'Custom variable';
 export const EMPTY_KEY = '---NO KEY---';
+export const MOUNT_PATH = '/opt/app-root/src';
```
Comment (andrewballantyne) [inline frontend/src/pages/notebookController/const.ts:3]: This is the same route as the [backend notebook workingDir](https://github.com/opendatahub-io/odh-dashboard/blob/main/backend/src/routes/api/notebooks/notebookUtils.ts#L79) right? Should we make some sorta comment / constant-ize the backend variable as well?

The idea here is keep them sync, no?

**Review by andrewballantyne — COMMENTED** [review]
One question @DaoDaoNoCode -- let me know what you think.

**Review by andrewballantyne — APPROVED** [review]
Awesome, putting it all together on one side of the fence works out great. 👍

File: frontend/src/pages/notebookController/const.ts (hunk 1 above)
Comment (samuelvl) [reply frontend/src/pages/notebookController/const.ts:3]: Yes, good idea

**Pull request merged**